driver/src/orderbook/streamed/orderbook.rs

```diff
@@ -145,4 +207,33 @@ mod tests {
         assert_eq!(orders.len(), 1);
         assert_eq!(orders[0].account_id, Address::zero());
     }
+
+    #[test]
+    fn test_write_read_recover_full_cycle() {
```

Comment:
Usually, we shouldn't be doing real I/O in a unit tests (we could argue this is a simple integration test, which maybe should be ignored by default like the KrakenAPI tests). 

Is there a simple way to mock the file system in memory in rust?

Reply:
Not that I'm aware :(

Reply:
Just a suggestion but there could be a second test that just serializes the orderbook to and from a `String` or `Vec<u8>` so that the file system isn't needed for the unit test, and then as @fleupold suggested keep this as a mini-integration test (like the Kraken API test) that is `#[ignore]`-d by default and can be manually run.

Reply:
Ya cool, I can ignore this test and make another one. Just wondering... if we `#[ignore]` the test... then what is it for or when is it executed?